### ISSUE TITLE
Audit component demos to make sure they are demonstrating ARIA properties 

### ISSUE THREAD

**Issue by blackbaud-johnly** [blackbaud/skyux2-docs#923]

**Comment by blackbaud-johnly** [comment]
https://github.com/blackbaud/skyux2-docs/issues/966 seems related to this.

**Comment by blackbaud-johnly** [comment]
I audited the demos and code samples for all components, and this is what I found:

Checkbox
•	The icon checkboxes include the `label` property, but the other checkboxes are missing the `labelledBy` property.
Data manager
•	This code sample and demo include the `name` property in the `SkyDataViewConfig`. Do we need anything else?
Dropdown
•	In the first code sample, the `sky-dropdown` element is missing the ‘label` property.
•	Also in the first code sample, the `sky-dropdown-item` element is missing the ‘ariaRole` property.
•	The second code sample doesn’t include either of those elements, and I don’t see either of those ARIA properties.
•	The `sky-dropdown-menu` element is not used in either code sample, so it’s `ariaLabelledBy` and `ariaRole` properties are not used in the code samples.
•	The demo does not have any of these ARIA properties in place.
Filter
•	The `ariaControls` and `ariaExpanded` properties are missing in all the demo files.
Flyout
•	The `ariaDescribedBy`, `ariaLabelledBy` and `ariaRole` properties are all used in one of the `.ts` files.
Lookup
•	The ariaLabelledBy` property is used in the `.html` file.
•	The `ariaLabel` property is missing in all the demo files.
•	Neither ARIA property is included in the documentation, but I created https://github.com/blackbaud/skyux2-docs/issues/992 to fix that.
Modal
•	The `ariaDescribedBy` property is used in one of the `.ts` files.
•	The `ariaLabelledBy` and `ariaRole` properties are missing in all the demo files.
Paging
•	The demo/code sample does not include the `pagingLabel` property on the `sky-paging` element.
Radio button
•	The 2 `sky-radio-group` elements each have `ariaLabelledBy` properties. (However, the placement of the elements in relation to their headers is different. Is this correct?)
•	The `sky-radio` elements for the icon radio buttons all have `label` properties.
Select field
•	The code sample uses 6 instances of the `ariaLabelledBy` property, but it should probably have 2 more instances in the *Disabled select fields* section.
•	The code sample does not use the `ariaLabel` property, but I think that’s probably OK.
Split view
•	The code sample uses the `ariaLabel` properties on both the sky-split-view-drawer and sky-split-view-workspace elements
•	The demo also uses the `ariaLabel` properties on both the sky-split-view-drawer and sky-split-view-workspace elements
Tabs
•	None of the code samples use the `ariaLabel` or `ariaLabelledBy` properties on the `sky-tabset` elements.
•	The demo does not use the `ariaLabel` or `ariaLabelledBy` properties on the `sky-tabset` elements.
Toggle switch
•	The code samples do not use the `ariaLabel` property on the `sky-toggle-switch` element.
•	The demo does not use the `ariaLabel` property on the `sky-toggle-switch` element.
Tokens
•	The demo/code sample does not use the `sky-token` element to populate the `sky-tokens` element, so it does not use the `ariaLabel` property. Should we use a different implementation or somehow apply ARIA outside of the `sky-token` element?
Vertical tabs
•	The code samples and demo do not use the `ariaControls` or `ariaRole` properties on the `sky-vertical-tab` elements and do not use the `ariaRole` property on the `sky-vertical-tabset` elements.
Wait
•	The demo/code sample does not use the `ariaLabel` property on the `sky-wait` element.

The following components don’t have SKY UX ARIA properties:
Action button
Alert
Autocomplete
Avatar
Back to top
Button
Card
Character count
Code
Code block
Colorpicker
Confirm – the code sample/demo do have `aria-haspopup` properties
Copy to clipboard
Country field
Data entry grid
Date pipe
Date range picker
Datepicker
Definition list
Email validation
Error
File attachment
Fluid grid
Form – the code samples do include `aria-label` and `aria-required` properties.
Format
Grid
Help inline
Icon
Infinite scroll
Inline delete confirmation
Inline form – the code sample includes `aria-label` properties.
Key info
Label
Link records
List overview
List filters
List paging
List toolbar
List view checklist
List view grid
Media queries
Navbar
Numeric
Page
Page summary
Phone field
Popover
Progress indicator
Repeater
Search
Sectioned form
Sort
Status indicator
Summary action bar
Text expand
Text expand repeater
Text highlight
Tile
Timepicker
Toast
Toolbar
Tree view – should this include third-party ARIA properties?
URL validation
Window
Wizard


I think we need to create create separate issues to update the demos and code samples. I'll need a developer to go through these and help decide which components need issues.

**Comment by Blackbaud-AlexKingman** [comment]
Thanks for doing all this @blackbaud-johnly ! We've moved your findings over to the story in ADO, where we can start tracking progress. Closing.
